### PR TITLE
Bump cirrus image with easier dependency management

### PR DESCRIPTION
**Pull request by lsm5**
Related:
https://github.com/containers/automation_images/issues/159
https://github.com/containers/automation_images/issues/161

Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by mheon** [comment]
All tests are red

**Comment by lsm5** [comment]
guess i need to tell CI to not download but install deps. Looking ..

**Comment by lsm5** [comment]
Let's wait until buildah and skopeo also pass with the same change.

**Comment by rhatdan** [comment]
@lsm5 @cevich is this still being worked?

**Comment by lsm5** [comment]
> @lsm5 @cevich is this still being worked?

it is

**Comment by vrothberg** [comment]
Friendly ping.  Is this PR still needed?

**Comment by cevich** [comment]
Dang, you'll have to remind me why this PR is bypassing `$PACKAGE_DOWNLOAD_DIR` to install packages at runtime.  Looking in c/automation_images, I see these packages are still being downloaded/saved into that directory.  Do you remember the reason we decided runtime-installs were a good idea here?

**Comment by cevich** [comment]
[Found it](https://github.com/containers/automation_images/issues/161)

> In cache_images/fedora_packaging script. It's installed via dependency resolution now, no need to download/cache it.

So likely the runtime install is a *noop*.  Which maybe begs the question, why bother with the `dnf` commands then?  Maybe it's worth removing them, since it will serve as a check on the image-build - ensuring the packages are indeed installed by dependency (as they should be).

**Comment by lsm5** [comment]
> Dang, you'll have to remind me why this PR is bypassing `$PACKAGE_DOWNLOAD_DIR` to install packages at runtime. Looking in c/automation_images, I see these packages are still being downloaded/saved into that directory. Do you remember the reason we decided runtime-installs were a good idea here?

IIRC, the idea was to avoid download packages and fetch them from fedora's own repos.

**Comment by lsm5** [comment]
@n1hility mind if I bother you about the Windows Smoke Test? It's failed quite a few times now.

**Comment by n1hility** [comment]
@lsm5 sure anytime. It looks like https://github.com/containers/automation_images/pull/177 needs to be rebased to pick up the windows images changes, and the new image update in this PR pulled in. Then it should work

File: .cirrus.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ env:
     UBUNTU_NAME: "ubuntu-2204"
 
     # Image identifiers
-    IMAGE_SUFFIX: "c6447802205601792"
+    IMAGE_SUFFIX: "c6534102392766464"
```
Comment (cevich) [inline .cirrus.yml:36]:
```suggestion
    IMAGE_SUFFIX: "c5841131699699712"
```

**Comment by cevich** [comment]
Aye, I rebased @lsm5 PR yesterday late.  That change hasn't made it here yet.  @vrothberg this failure is entirely expected.  Things get complicated when there are multiple CI VM Image build PRs running in parallel.  Not much we can do to simplify that.

**Comment by cevich** [comment]
I restarted the `bud podman fedora-37 root host` task.  Let's get this merged :smiley:

**Comment by lsm5** [comment]
@cevich @containers/podman-maintainers ready for review.

**Review by vrothberg — APPROVED** [review]
LGTM

**Pull request merged**